WeConnect_RestAPI/WeConnect_RestAPI/api/we_connect/business.py

```diff
@@ -0,0 +1,16 @@
+import datetime
+from WeConnect_RestAPI.api.restplus import api
+
+class WeConnect(object):
+    
+
+    def __init__(self):
+        '''Constructor method, will have dictionaries in a list'''
+        self.users_counter = 0
```

Comment:
Why have this? Can't you just get the length of the users list?

Reply:
Performance I guess. Checking the value of the counter then adding 1 instead of checking for the length of the list each time.